dGame/dComponents/ControllablePhysicsComponent.cpp

```diff
@@ -11,6 +11,7 @@
 #include "CDClientManager.h"
 #include "EntityManager.h"
 #include "Character.h"
+#include "dZoneManager.h"
```

Comment:
My only concern is if this was the best way to be able to check if we are in a zone we don't want t save in from the component.